### PR TITLE
[SDK-208][Node.js SDK] [EOS-54] Update README to mention Schema 2.2 changes

### PR DESCRIPTION
**Pull request by igor-alekseev-dev**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-208](https://incountry.atlassian.net/browse/SDK-208)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [x] Please put the target release vehicle this needs to be bundled with.
- [x] Did you increment the version in package.json? What is the new version?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,39 +25,70 @@ Usage
...
+    clientId?: string;     // Required when using oAuth authorization, can be also set via environment variable INC_CLIENT_ID
+    clientSecret?: string; // Required when using oAuth authorization, can be also set via environment variable INC_CLIENT_SECRET
+    authEndpoints?: {      // Custom endpoints regional map to use for fetching oAuth tokens
+      default: string;
+      [key: string]: string;
+    };
+  };
+
+  endpoint?: string;       // Defines API URL
+  encrypt?: boolean;       // If false, encryption is not used. If omitted is set to true.
```
Comment (RommeroInCountry) [inline README.md:42]: If false, encryption is not used. Defaults to true.

**Review by artemgolozubov — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -249,57 +306,97 @@ const writeResult = await storage.write(country, record);
...
+```typescript
 batchResult = await storage.batchWrite(country, records);
 ```
 
 ### Reading stored data
 
-Stored record can be read by `key` using `read` method. It accepts an object with two fields: `country` and `key`.
-It returns a `Promise` which resolves to `{ record }`  or  `{ record: null }` if there is no record with this `key`.
+Stored record can be read by `recordKey` using `read` method. It accepts an object with two fields: `country` and `recordKey`.
+It returns a `Promise` which resolves to `{ record }`  or  rejects Promise if there is no record with this `recordKey`.
```
Comment (RommeroInCountry) [inline README.md:346]: `read` returns a `Promise` which resolves to `{ record }`  or is rejected if there are no records for the given `recordKey`.

**Review by RommeroInCountry — CHANGES_REQUESTED** [review]

**Review by artemgolozubov — APPROVED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -241,65 +328,89 @@ const record = {
...
 
-```javascript
-/**
- * @param {string} countryCode Country code
- * @param {string} recordKey
- * @param {object} [requestOptions]
- * @return {Promise<{ record: Record|null }>} Matching record
- */
-async read(countryCode, recordKey, requestOptions = {}) {
+#### Created and updated dates fields
```
Comment (RommeroInCountry) [inline README.md:362]: simply "Date fields" would do

File: README.md [hunk 4]
```diff
@@ -241,65 +328,89 @@ const record = {
...
+
+Fields `createdAt` and `updatedAt` store dates set by InCountry storage server. `createdAt` indicates when the data was first time processed in the target country. `updatedAt` indicates when last write operation finished.
```
Comment (RommeroInCountry) [inline README.md:364]: Use `createdAt` and `updatedAt` fields to access date-related information about records. `createdAt` indicates date when the record was initially created in the target country. `updatedAt` shows the date of the latest write operation for the given `recordKey`

**Review by RommeroInCountry — CHANGES_REQUESTED** [review]

**Review by artemgolozubov — APPROVED** [review]
LGTM

**Review by VladimirLevitskiy — APPROVED** [review]
LGTM

**Review by RommeroInCountry — APPROVED** [review]
LGTM

**Pull request merged**